### ISSUE TITLE
Enable key binding for toggle

### ISSUE THREAD

**Issue by mattjgarland** [mupchrch/split-diff#66]
I'd like toggle split-diffing on and off for a file, so it would be a great enhancement to expose the toggle for key binding.

**Comment by mupchrch** [comment]
Hi someone actually submitted a pull request for this #64 but I was unsure what the keybinding should be. I didn't want it to conflict with other packages too badly. Do you have any ideas?

**Comment by mattjgarland** [comment]
ctrl-alt-g? (split) It doesnt matter to me much since I'll be reassigning it my particular system (I'd like my git commands to be cmd-5 -> cmd-9). 

There might be another way around it, however. I *thought* I noticed that first use of ctr-alt-n for split-diff:next-diff actually toggles the diff on and selects the first diff (I'll check this when I get home). Then when I remapped the key it lost the toggle behavior and would only work if the toggle was already on. Is there a way to make sure next-diff always toggles the diff pane on if it is not on, even if the key is remapped? If there is, I would not need to have the key mapping for toggling exposed.
